stdlib/http/src/main/ballerina/http/annotation.bal

```diff
@@ -134,12 +134,12 @@ public type WebSocketUpgradeConfig record {|
 # Configures the authentication scheme for a service or a resource.
 #
 # + enabled - Specifies whether authentication is enabled
-# + authnHandlers - Array of authentication handlers
-# + scopes - Array of scopes
+# + authnHandlers - Array of authentication handlers or Array of array of authentication handlers
```

Comment:
For clarify of API documentation, can we also state the usage of array and array of arrays separately? Same for scopes.